### PR TITLE
Include IfcURIReference in the IfcSimpleValue SELECT

### LINKED ISSUE #619: Add IfcURIReference to the IfcSimpleValue SELECT type
To be able to use an IfcURIReference as an IfcPropertySimpleValue in a Pset.

### PR DESCRIPTION
**Pull request by larswik**
Fixes #619

### CODE REVIEW

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Types/IfcSimpleValue/Documentation.md [hunk 1]
```diff
@@ -14,10 +14,13 @@ SELECT
...
+*  _IfcURIReference_: Defined type of simple type STRING to represent a URI reference. 
 
 > _HISTORY&nbsp; New type in IFC2x._
 
 { .change-ifc2x4}
 > _IFC4 CHANGE&nbsp; Items _IfcDateTime_,
       _IfcDate_, _IfcTime_, _IfcDuration_,
       _IfcTimeStamp_ added._
+{ .change-ifc4x4}
+> _IFC4 CHANGE&nbsp; Item _IfcURIReference_added._
```
Comment (pjanck) [inline IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Types/IfcSimpleValue/Documentation.md]: I don't know how the rendering procedure works, but I assume we need a space before _added_:

```suggestion
> _IFC4 CHANGE&nbsp; Item _IfcURIReference_ added._
```

**Review by pjanck — APPROVED** [review]
lftm

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Types/IfcSimpleValue/Documentation.md (hunk 1 above)
Comment (larswik) [reply IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Types/IfcSimpleValue/Documentation.md]: > I don't know how the rendering procedure works, but I assume we need a space before _added_:

Yes, I forgot to ask. I am not sure either on how to encode these things so good if @SergejMuhic can check this.

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Types/IfcSimpleValue/Documentation.md [hunk 2]
```diff
@@ -14,10 +14,13 @@ SELECT
...
 *  _IfcTimeStamp_: Defined type of simple type INTEGER to represent a point in time by seconds elapsed since 1970. 
+*  _IfcURIReference_: Defined type of simple type STRING to represent a URI reference. 
 
 > _HISTORY&nbsp; New type in IFC2x._
 
 { .change-ifc2x4}
 > _IFC4 CHANGE&nbsp; Items _IfcDateTime_,
       _IfcDate_, _IfcTime_, _IfcDuration_,
       _IfcTimeStamp_ added._
+{ .change-ifc4x4}
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Types/IfcSimpleValue/Documentation.md]: Can lose this.

Comment (SergejMuhic) [inline IFC4x3/Sections/Resource definition data schemas/Schemas/IfcMeasureResource/Types/IfcSimpleValue/Documentation.md]:
```suggestion
```

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**